shanoir-ng-front/src/app/shared/components/entity/entity-list.component.abstract.ts

```diff
@@ -125,7 +125,11 @@ export abstract class EntityListComponent<T extends Entity> implements OnDestroy
                     this.getService().delete(entity.id).then(() => {
                         this.onDelete.next({entity: entity});
                         this.table.refresh().then(() => {
-                            this.consoleService.log('info', 'The ' + this.ROUTING_NAME + ' n°' + entity.id + ' sucessfully deleted');
+                            if (this.ROUTING_NAME == 'examination') {
+                                this.consoleService.log('info', 'The ' + this.ROUTING_NAME + ' n°' + entity.id + ' has sucessfully started to delete. Check the job page to see its progress.');
```

Comment:
This should be done in entity.abstract.service.ts too (so that you have the specific message on exam detail too)